undertow/src/main/resources/schema/wildfly-undertow_2_0.xsd

```diff
@@ -199,6 +199,7 @@
         <xs:attribute name="name" use="required" type="xs:string"/>
         <xs:attribute name="alias" use="optional" type="xs:string"/>
         <xs:attribute name="default-web-module" use="optional" type="xs:string" default="ROOT.war"/>
+        <xs:attribute name="custom-response-code" use="optional" type="xs:int"/>
```

Comment:
custom-response-code is not a great name for this, it should be default-response-code